## 1 Introduction
Yong C H , et al. described a draft whole-genome assembly of the D. zibethinus Musang King cultivar and did phylogenetic researches. They also compare transcriptome data between different plant organs and different cultivars.   
  
  
This project aims to reproduce this research using the provided data. With this project, student can learn useful skills and experience the real research process.

## 2 Data used in the paper 
* DNA reads generated by PacBio and Illumina. From Musang King fruit stalk. Illumina reads later be improved using CHiCAGO and Hi-C.
* RNA reads generated by Illumina. From fruit arils of Musang King, Monthong and Puang Manee. And from stem, leaf and root of a non-fruit group.
* Quantification of sulfur-containing volatiles and ethylene.
* (share and improve this info with others in the group)

## 3 Analyses in the paper
(share and improve this info with others in the group)
### Assembly of DNA
First run CANU to assembly the PacBio reads and apply Redundans to reduce fragmentation. But result no good, so use FALCON instead, and Arrow for polish. Use Pilon for final correction, based on Illumina short reads.

### Genome size and ploidy
Jellyfish implement k-mer distribution analysis on Illumina short reads and show the genome size of 738Mb and diploidy.

### Scaffolding
Align shotgun and CHiCAGO library reads with a de novo assembly using SNAP, then use HiRise for scaffolding. Do the same in Hi-C library reads, and combine the two results.

### Assembly of RNA-seq
Preprocess data with trim_galore. Align reads to the reference genome with HiSat.  Produce non-redundant and reference-guided assembly with StringTie.

### Genome annotation
Use RepeatModeler for repeat detection, Maker for gene prediction (evidence from Arabidopsis, grape, rice, and soybean), Blast2GO for genome ontology, BLASTP to incorporate sequence homology with Arabidopsis genes and InterProScan for functional domain.

### Short-read alignment and SNP calling
Align the preprossed RNA-seq reads to the assembly with BWA-MEM. Sort alignment with SAMtools. Remove duplicates with Picardtools and infer SNPs with FreeBayes.

### Molecular phylogenetic analysis
Cluster the gene for 11 plant species using OrthoMCL (see Figure 2 from original article). Later will count and compare the number of gene members to test gene family expansion. Coden aligned gene for each family with MUSCLE and curate the alignment with Gblocks. Implement phylogenetic analysis using BEAST.

### Whole-genome duplication analysis
Detect detect syntenic blocks and calculate Ks with MCScan and CoGe. Estimate the phylogenetic tree and divergence times of the paralogous and orthologous genes in durian and G. raimondii,cacao and Arabidopsis with BEAST.

### Transcriptome analysis
Count aligned short-reads of RNA-seq against the predicted gene models. Test differential expression and shrink data with DESeq. Remove hidden variables with sva package.  Use these data for Gene Set Enrichment Analysis with fgsea.


## 3 What to do in this project (according to LabManual)

#### Data on hand: 

illumina_data: scaffold

pacbio_data: scaffold

transcriptome : trimmed & untrimmed, both scaffold

* Genome assembly of PacBio reads - CANU, `Redundans, Arrow`
  * Assembly with different parameters (do at last/ meanwhile)
  * Genome size and ploidy(while waiting for assembly result) - Jellyfish
* Correct the assembly with Illumina reads - Pilon
* Assembly quality assessment - QUAST and CANU¡¯s N50
  * Assembly evaluation with more than one method
* Structural and functional annotation - RepeatModeler, Maker, Blast2GO, BLASTP, InterProScan
* Reads preprocessing: trimming + quality check (before and after) - trim_galore
<br/>(prictice with part of untrimmed data, then continue with trimmed data)
* Transcriptome assembly - `Tophat,` Trinity
* Differential expression analyses - HTSeq, DESeq, fgsea
  * Deeper analyses of differential expression analyses
* Biological interpretation of the results - combine everything
  * Do research based on current status - statistics or what else missing



## 4 Expected result
Largely the result should be similar to the original paper. There could be slightly different, since the author says CANU didn't perform as good as FALCON, plus we are using part of data.

## 5 Data management
Make link to the raw data and save the results. For each task start a new folder, name the folder indicating the task name. For each of the task, build folder by date in case to do analysis again. Within the folder, name the result files. If possible, record which file come from what action on which file.

## 6 Time scale
Follow the timeline of Metagenomics project in Appendix III of LabManual. If have time, do extra tasks. If not, skip extra tasks.



## 7 References
Teh B T , Lim K , Yong C H , et al. The draft genome of tropical fruit durian (Durio zibethinus)[J]. Nature Genetics, 2017.



